packages/website/index.html

```diff
@@ -12,14 +12,14 @@
       <meta property="og:description" content="Collection of essential React Hooks Utilities." />
       <meta charSet="UTF-8" />
       <meta name="keywords" content="reactuse,react" />
-      <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
       <meta name="description" content="Collection of essential React Hooks Utilities." />
       <meta name="viewport" content="width=device-width, initial-scale=1.0" />
       <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
     </head>
     
     <body>
-      <div id="main" class="markdown-body"></div>
+      <div id="main"></div>
```

Comment:
<img width="745" alt="image" src="https://github.com/childrentime/reactuse/assets/58261676/67c47a8e-06e3-43a9-b887-043b7177fe2c">
it will cause